examples/image_bright_regions_gallery_example.py

```diff
@@ -0,0 +1,95 @@
+
+# coding: utf-8
+
+"""
+====================================================
+Manipulating Map image data - finding bright regions
+====================================================
+
+This example shows how you can do basic image processing on SunPy map image data.
+In this example, we try to find the brightest regions in an AIA image.
+Then count the approximate number of regions of interest.
+"""
+
+##############################################################################
+# First, import the modules we will need:
+
+import matplotlib.pyplot as plt
+import numpy as np
+from scipy import ndimage
+import sunpy.map
+from sunpy.data.sample import AIA_193_IMAGE
+
+##############################################################################
+# Now, we create a SunPy Map object from an AIA FITS file.
+
+aiamap = sunpy.map.Map(AIA_193_IMAGE)
+
+##############################################################################
+# Let's plot the map.
+# The aiamap normalizes the range using an asinh function.
+# This show features in an image with high dynamic range more clearly.
+
+plt.figure()
+aiamap.plot()
+plt.colorbar()
+plt.show()
+
+##############################################################################
+# Now we want to find the brightest regions in this image.
+# We start by finding the maximum value in the image data.
+
+data_max = aiamap.max()
+
+##############################################################################
+# Now we want to make a mask, which tells us which regions are bright.
+# We choose the criterion that the data should be at least 5% of the maximum value.
+# Pixels with intensity values greater than this are included in the mask, while all other pixels are excluded.
+
+mask = aiamap.data < data_max*0.05
+
+##############################################################################
+# Mask is a `boolean` array. It can be used to modify the original map object without modifying the data.
+# Once this mask attribute is set, we can plot the image again.
+
+aiamap.mask = mask
+plt.figure()
+aiamap.plot()
+plt.colorbar()
+plt.show()
```

Comment:
How is it done in other examples with lots of figures? 

Do we want all of them to show up or for it to be done bit by bit?